### PR TITLE
Add testing.add_misc_table()

### LINKED ISSUE #260: Add testing.add_misc_table()
It would be a good idea to add `testing.add_misc_table()` or expand `testing.add_table()` as discussed in https://github.com/audeering/audformat/pull/255#discussion_r934238298

### PR DESCRIPTION
**Pull request by frankenjoe**
Closes #260 

image

In `testing.add_table()` we change to:

image

### CODE REVIEW

File: audformat/core/testing.py [hunk 1]
```diff
@@ -34,6 +34,52 @@
...
+    To create a specific set of columns use ``columns``.
+
+    Args:
+        db: a database
+        table_id: ID of table that will be created
+        index: index object
+        columns: a list of scheme_ids or a dictionary with column names as
+            keys and tuples of ``(scheme_id, rater_id)`` as values. ``None``
+            values are allowed
+        p_none: probability to draw invalid values
```
Comment (hagenw) [inline audformat/core/testing.py:65]: This I do not get completely, how can we assign values that do not match the scheme without getting an error?

Comment (frankenjoe) [reply audformat/core/testing.py:65]: With `invalid` we refer to `None` values. I will rephrase.

Comment (frankenjoe) [reply audformat/core/testing.py]: done

Comment (hagenw) [reply audformat/core/testing.py]: Ah, this makes indeed more sense, thanks.

**Pull request merged**